### ISSUE TITLE
Error: Invalid use of type "undefined" as a Firestore argument.

### ISSUE THREAD

**Issue by pollo17385** [jloosli/node-firestore-import-export#34]
### Expected behavior
Make an normal import of the TimeStamp type.

### Actual behavior
When trying to import a export made on 0.10.3 I get this error:

Error: Invalid use of type "undefined" as a Firestore argument.
Error: Invalid use of type "undefined" as a Firestore argument.
    at Object.customObjectError (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\validate.js:168:16)
    at Serializer.encodeValue (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:145:26)
    at Serializer.encodeFields (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:53:34)
    at Serializer.encodeValue (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:138:44)
    at Serializer.encodeFields (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:53:34)
    at Serializer.encodeValue (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:138:44)
    at Serializer.encodeFields (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\serializer.js:53:34)
    at Function.fromObject (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\document.js:141:53)
    at WriteBatch.set (C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\node_modules\@google-cloud\firestore\build\src\write-batch.js:241:54)
    at C:\Users\Pc\AppData\Roaming\npm\node_modules\node-firestore-import-export\dist\lib\import.js:66:19

### Steps to reproduce the behavior
Make an export of the DB and then try to import it.

**Comment by chwasifjameel** [comment]
Facing the same issue. Don't know why!

**Comment by Endanke** [comment]
I think the issue originated from the unserializeSpecialTypes function from the helpers.ts file.
There's a check for the data type, at row 61:
`  if (rawValue instanceof Object) {`
And it is not equal if the property is not an object. In my case, I have a list of companies and they have a 'name' field which is a string, so the condition fails.

**Comment by wujekbogdan** [comment]
I have the same problem. I exported a single collection.

``` bash
firestore-export --accountCredentials ./env/serviceAccountKey.production.json --nodePath teams --backupFile teams.json
```

Then I tried to import it:
``` bash
firestore-import --accountCredentials ./env/serviceAccountKey.staging.json --nodePath teams --backupFile teams.json
```

And I'm getting the same error.

node  v10.12.0
macOS 10.13.6

**Comment by jefrydco** [comment]
Hi @jloosli , would you like to take a look this issue, the error is still occured :(

**Comment by csimoes1** [comment]
I am seeing the same problem:
UnhandledPromiseRejectionWarning: Error: Invalid use of type "undefined" as a Firestore argument

**Comment by csimoes1** [comment]
I have checked in a pull request with a fix for this issue.  Please review here https://github.com/jloosli/node-firestore-import-export/pull/35

**Comment by jloosli** [comment]
Resolved with #35
